### ISSUE TITLE
Consider migrating to Jekyll for easy maintenance

### ISSUE THREAD

**Issue by RaoOfPhysics** [Meta-Paleo/meta-paleo.github.io#5]
Hey @Protohedgehog,

I'm guessing you used Rmarkdown to build the website using these instructions? http://nickstrayer.me/RMarkdown_Sites_tutorial/

Perhaps you would consider migrating the website to Jekyll instead, given that it works closely with GitHub pages: https://help.github.com/articles/using-jekyll-as-a-static-site-generator-with-github-pages/

It's really easy to set up -- I'm happy to help! -- and you don't need any special software on your end to work with it. If you _do_ want to test the website before you push it to GitHub, you can install Jekyll locally, but in theory you can work directly (and only!) with `.md` files for all your content. Jekyll also simplifies any changes people want to make to your website, as they can edit the Markdown files straight in GitHub and you can merge it without anyone having to run `Rmarkdown::render_site()` from R.

Just a thought. Let me know if you think it's worth doing and if you would like my help. :)

**Comment by Protohedgehog** [comment]
Yep, and also from here: https://gupsych.github.io/acadweb/project.html

What would the benefits of migrating the website be..?

**Comment by RaoOfPhysics** [comment]
You don't change any HTML files directly, you merely edit the `.md` files and Jekyll at GitHub's end takes care of converting it to HTML. So, the overhead for contributors is low. For the PR I made, for example, there would only be one change to the footer file alone. You get a cleaner commit history as well.

**Comment by Protohedgehog** [comment]
Ah, this is what I've been doing anyway, I think, simply in RStudio. Seems to be working okay.. :)

**Comment by Protohedgehog** [comment]
@RaoOfPhysics just visiting this again now after way too long. I hope you're doing splendid!

Do you think this is still something worth doing? Atm I can write the site content in markdown, and then just build the site in RStudio to create all the HTML files, and then push to make it all live online. Is Jekyll still worth looking into..?

**Comment by RaoOfPhysics** [comment]
Doing well. :) Are you travelling?

Yeah, I still think it's better to keep just the Markdown files under version control and let GitHub handle the generation of the HTML files via Jekyll.

At the moment, if you go through your commit history, you effectively have a duplicate set of commits: `foo.Rmd` and `foo.html` are both changed and pushed to GitHub when you make a change in `foo.Rmd` and tell RStudio to build `foo.html`.

Instead, you could have only `foo.md` in your commits, with `foo.html` (or, even better in my eyes, `foo/index.html`) being generated online.

Unless you're using any `R` code in your website to make plots or add interactivity, you're better of using plain Markdown rather than R Markdown.

**Comment by Protohedgehog** [comment]
Gah, sorry for missing this @RaoOfPhysics. Yes, traveling a bit ;)

OK, so, migrating to Jekyll. Reckon we should do this? :)

**Comment by Protohedgehog** [comment]
Consider my knowledge of this to be close to, or less than, zero.

**Comment by RaoOfPhysics** [comment]
I'd be up for it, but not in any reasonable time scale, unfortunately. How active is the website and when do you think you might want to work on the migration?

**Comment by Protohedgehog** [comment]
Understandable. I just spoke about this with @msantolini and he agreed the migration would be a good idea too.

The website is, er, variably active. I just updated it and you're now on it! https://meta-paleo.github.io/index.html 

Congrats on the citation ;)
